app/pb_kits/playbook/pb_card/_card.jsx

```diff
@@ -14,10 +13,54 @@ type CardPropTypes = {
   },
   padding?: 'none' | 'xs' | 'sm' | 'md' | 'lg' | 'xl',
   selected?: Boolean,
-  shadow?: 'none' | 'shallow' | 'default' | 'deep' | 'deeper' | 'deepest',
+  shadow?: 'none' | 'deep' | 'deeper' | 'deepest',
   dark?: Boolean,
 }
 
+type CardHeaderProps = {
+  children: Array<React.ReactNode> | React.ReactNode,
+  className?: String,
+  padding?: 'none' | 'xs' | 'sm' | 'md' | 'lg' | 'xl',
+  color?: 'category_1' | 'category_2' | 'category_3' | 'category_4' | 'category_5' | 'category_6' | 'category_7' | 'category_8' |
+          'category_9' | 'category_10' | 'category_11' | 'category_12' | 'category_13' | 'category_14' | 'category_15' | 'category_16' |
+          'category_17' | 'category_18' | 'category_19' | 'category_20' | 'category_21',
```

Comment:
Thoughts about making this `category: 1` instead. I feel like that's closer to what we're doing with spacing